src/redux/sagas.js

```diff
@@ -13,6 +18,18 @@ function* getSpendingsWorker(data) {
   }
 }
 
+function* deleteSpendingWorker(data) {
+  try {
+    yield call(spendings.deleteSpending, data.payload.id);
+    yield put(deleteSpendingSuccess(data.payload.id));
+  } catch {
```

Comment:
`catch (error)`. Also pass `error.message` to `deleteSpendingError` action-creator

Reply:
> `catch (error)`. Also pass `error.message` to `deleteSpendingError` action-creator

I have done it, but I'm not sure that it's right